Formula/immudb.rb

```diff
@@ -0,0 +1,41 @@
+class Immudb < Formula
+  desc "Lightweight, high-speed immutable database"
+  homepage "https://www.codenotary.io"
+  url "https://github.com/codenotary/immudb/archive/v0.6.0.tar.gz"
+  sha256 "2bed8d984767c970b8e0dc15a89db5737e909b3052489debd8db47c9ea3906ec"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "all"
+    bin.install %w[immudb immuclient immugw immuadmin]
+  end
+
+  test do
+    pids = []
+    pids << fork do
+      exec bin/"immudb"
+    end
+    pids << fork do
+      exec bin/"immugw"
+    end
+    sleep 1
+
+    system bin/"immuclient", "safeset", "hello", "world"
+    assert_match "world", shell_output("#{bin}/immuclient safeget hello")
+
+    command = %W[
+      curl -s -XPOST -d '#{JSON.generate({ "key" => Base64.encode64("hello") })}'
+      http://localhost:3323/v1/immurestproxy/item/safe/get
+    ]
+    response = shell_output(command.join(" "))
+    assert_equal "world", Base64.decode64(JSON.parse(response)["value"])
+
+    assert_match "Uptime", shell_output("#{bin}/immuadmin stats -t")
+  ensure
+    pids.each do |pid|
```

Comment:
No need to do this, brew will do this automatically these days

Reply:
Awesome!